docs/authentication/saml/jit-provisioning.mdx

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Does this require its own guide or could this be added as a section to /saml/overview right above the "[Sync user attributes during sign in](https://clerk.com/docs/pr/1114/authentication/saml/overview#sync-user-attributes-during-sign-in)" section ?

I feel like 1. It's basically a concept that is inextricably tied to SAML and the explanation of how we do SAML so it'd make sense to be in the overview, and 2. it'd be more easily found there

Reply:
That's not a bad idea, @chanioxaris what do you think?

Reply:
Even if this page doesn't have much content for the time being, JIT provisioning is one of the important features that comes with SAML that existing and potential customers are greatly interested in. So i would say it deserves it's own page to be explicit and mostly about be SEO friendly and easily discoverable.

We should also move to this new page the [Sync user attributes during sign in](https://clerk.com/docs/authentication/saml/overview#sync-user-attributes-during-sign-in) section from the overview page, as describes how JIT provisioning works during sign in

Reply:
Got it, thanks for the input 👍